src/Abc.Zebus/Core/BusMessageLogger.cs

```diff
@@ -24,23 +24,22 @@ public BusMessageLogger(Type loggerType)
 
         public BusMessageLogger(string loggerFullName)
         {
-            _logger = LogManager.GetLogger(typeof(BusMessageLogger).Assembly, loggerFullName);
+            _logger = ZebusLogManager.GetLogger(loggerFullName);
 
-            // Instances of BusMessageLogger are static, no need to unsubscribe from these events
-            _logger.Logger.Repository.ConfigurationChanged += (sender, args) => UpdateLogConfig();
-            _logger.Logger.Repository.ConfigurationReset += (sender, args) => UpdateLogConfig();
+            // Instances of BusMessageLogger are static, no need to unsubscribe from this event
```

Comment:
That's true but the `BusMessageLogger` is public. There could exist some non-static instances in some obscure codebase.

**suggestion:** Remove `_logDebugEnabled` and `_logInfoEnabled` fields, they are probably premature optimizations.

Reply:
That's true, and I think this class should have been internal. Maybe I should just make it internal? 🤔 

The fields are not a premature optimization, we introduced them because the perf difference was noticeable.